src/util/SplitMessage.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function splitMessage(text, { maxLength = 1950, char = '\n', prepend = '', append = '' } = {}) {
   if (text.length <= maxLength) return text;
-  const splitText = text.split(char);
-  if (splitText.length === 1) throw new Error('Message exceeds the max length and contains no split characters.');
+  let splitText = text.split(char);
+  if (splitText.length === 1) splitText = splitText.join('').match(new RegExp(`[\\s\\S]{1,${maxLength}}\b`, 'g'));
```

Comment:
Not sure if I'm a fan of this fallback, as it would likely ruin any formatting someone has created - which isn't really preferable to failing, IMO.
Even so, `splitText.join('')` is unnecessary - just use `text`.